### ISSUE TITLE
Docs site mixed content (HTTP vs HTTPS)

### ISSUE THREAD

**Issue by bzier** [scipy/scipy#8934]
The SciPy documentation [site](https://docs.scipy.org/doc/) does not load securely (in Google Chrome) due to mixed content (HTTPS vs HTTP). This seems to have been resolved in the Sphinx repo in this [commit](https://github.com/sphinx-doc/sphinx/commit/f9250e198b5dab4ab9ff9a489ffdf723d873451c#diff-efa15804d6baa4dbab13c1d4788865c1) and in the SciPy Sphinx Theme repo in this [commit](https://github.com/scipy/scipy-sphinx-theme/commit/0eaef7ac75f6181c5b8d6f6608a46688fcf202ed#diff-d92810ca57a19738154e282c2426f6f1). However, it doesn't appear that the doc site has been updated with these changes.

![image](https://user-images.githubusercontent.com/714528/41494082-6a9ff910-70c3-11e8-97a4-88dc0cb7facf.png)

I couldn't find another issue with this, but if I overlooked it, feel free to refer me to the duplicate. Thanks.

**Comment by rgommers** [comment]
I suggest we update the latest release. We can't really go and update all the old stuff, that would take forever. Looks like that still works, just with a warning about security from Chrome, so I don't really want to rebuild a gigabyte worth of doc pages.

**Comment by bzier** [comment]
Yes, the site still _works_ just fine. Chrome blocks the insecure content by default, but that seems to just be specific fonts, so it isn't critical content. Because the 'insecure' content isn't loaded, Chrome still shows the site as 'secure' with the green lock. However, if you choose to unblock the insecure content, then Chrome changes to show the site as 'not secure' with the red exclamation triangle and the red strike-through of 'https' in the url bar.

You can do with it whatever you like, I just wanted to point it out after I bumped into it yesterday. I saw that the source had been updated a couple years ago. The insecure reference is coming out of the single file `spc-bootstrap.css`. Wouldn't it be possible to update that file at `https://docs.scipy.org/doc/_static/css/spc-bootstrap.css` without having to rebuild the entire docs site?

**Comment by rgommers** [comment]
Thanks for the details @bzier. Yes replacing that file is easy, however there are lots of copies floating around, e.g. see https://docs.scipy.org/doc/scipy-0.19.1/reference/_static/css/. And I'm not sure if we go around overwriting really old copies, whether or not we will break anything. Hence I think just fixing the instance that you just pointed out would be best.

**Comment by bzier** [comment]
Ah, I see. I didn't realize that it was scattered elsewhere.
